datafusion/core/src/datasource/file_format/parquet.rs

```diff
@@ -848,7 +859,13 @@ fn spawn_column_parallel_row_group_writer(
             mpsc::channel::<ArrowLeafColumn>(max_buffer_size);
         col_array_channels.push(send_array);
 
-        let task = SpawnedTask::spawn(column_serializer_task(recieve_array, writer));
+        let reservation =
+            MemoryConsumer::new("ParquetSink(ArrowColumnWriter)").register(pool);
```

Comment:
Does the name used have any uniqueness significance? Because we could identify each writer by field name.